code/modules/mob/living/simple_animal/hostile/nanotrasen.dm

```diff
@@ -46,8 +51,6 @@
 	minimum_distance = 5
 	casingtype = /obj/item/ammo_casing/c45
 	projectilesound = 'sound/weapons/gunshot.ogg'
-	loot = list(/obj/item/gun/ballistic/automatic/pistol/m1911,
```

Comment:
Can't do that, because the nanotrasen operatives that I based my hostiles entites off of for cybersun when then be dropping firearms which would be a substantial amount.